### DISCUSSION TITLE
Can I use Curobo in Isaac Gym?

### DISCUSSION THREAD

**Discussion by branvu** [NVlabs/curobo#84] [Software Q&A] [answered]
Is it possible to use Curobo in Isaac gym? Or is it mostly tied to Isaac Sim?

Thanks!
Brandon

**Comment by balakumar-s** [comment, marked as answer]
You can use it with isaac gym. 

cuRobo is a python library and can be used with any simulator or real robot. 

We implemented an example interface to isaac sim so that users can test it out.
